Assets/HoloToolkit/Input/Scripts/InputEventData/PointerInputEventData.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This fixes a bug.  When UnityUI events were flowing, they weren't correctly "pointer-stealing" between 6DoF pointing ray and gaze.  By attaching InputSource and SourceId to those events, too, "pointer-stealing" works correctly.